app/src/main/java/com/team41/boromi/book/SearchFragment.java

```diff
@@ -33,27 +24,19 @@ public SearchFragment() {
    * Use this factory method to create a new instance of this fragment using the provided
    * parameters.
    *
-   * @param param1 Parameter 1.
-   * @param param2 Parameter 2.
    * @return A new instance of fragment SearchFragment.
    */
   // TODO: Rename and change types and number of parameters
-  public static SearchFragment newInstance(String param1, String param2) {
+  public static SearchFragment newInstance() {
     SearchFragment fragment = new SearchFragment();
     Bundle args = new Bundle();
```

Comment:
I am a little confused on what this does. Does calling new Bundle() grab the arguments for you. Or is it actually just an empty bundle object?

Reply:
No, here you would pass in any arguments into your bundle where it will be grabbed in the onCreate method. Here, we don't pass in anything but could in the future.

Reply:
I see, so currently it's not doing anything, but more of a placeholder